test/unit-tests/components/views/rooms/RoomListPanel/RoomList-test.tsx

```diff
@@ -44,19 +39,15 @@ describe("<RoomList />", () => {
     });
 
     it("should render a room list", () => {
-        const { asFragment } = render(<RoomList vm={vm} />, withClientContextRenderOptions(matrixClient));
+        const { asFragment } = render(<RoomList vm={vm} />, {
+            wrapper: ({ children }) => (
+                <MatrixClientContext.Provider value={matrixClient}>
+                    <VirtuosoMockContext.Provider value={{ viewportHeight: 600, itemHeight: 56 }}>
+                        <>{children}</>
+                    </VirtuosoMockContext.Provider>
+                </MatrixClientContext.Provider>
+            ),
+        });
         expect(asFragment()).toMatchSnapshot();
     });
-
-    it.each([
-        { shortcut: { key: "F6", ctrlKey: true, shiftKey: true }, isPreviousLandmark: true, label: "PreviousLandmark" },
-        { shortcut: { key: "F6", ctrlKey: true }, isPreviousLandmark: false, label: "NextLandmark" },
-    ])("should navigate to the landmark on NextLandmark.$label action", ({ shortcut, isPreviousLandmark }) => {
-        const spyFindLandmark = jest.spyOn(LandmarkNavigation, "findAndFocusNextLandmark").mockReturnValue();
-        const { getByTestId } = render(<RoomList vm={vm} />, withClientContextRenderOptions(matrixClient));
-        const roomList = getByTestId("room-list");
-        fireEvent.keyDown(roomList, shortcut);
-
-        expect(spyFindLandmark).toHaveBeenCalledWith(Landmark.ROOM_LIST, isPreviousLandmark);
```

Comment:
Why is this test going away?

Reply:
Ha, I have no idea why this was deleted and it is indeed broken. Will fix, sorry about that.